packages/ramp-core/src/fixtures/export-v1/appbar-button.vue

```diff
@@ -15,15 +12,17 @@
     </appbar-button>
 </template>
 <script lang="ts">
-import { Vue, Prop } from 'vue-property-decorator';
+import { defineComponent } from 'vue';
 
-export default class ExportV1AppbarButtonV extends Vue {
-    @Prop({ default: { colour: 'auto' } }) options!: { colour: string };
-
-    onClick() {
-        this.$iApi.panel.toggle('export-v1-panel');
+export default defineComponent({
+    name: 'ExportV1AppbarButtonV',
+    props: ['options'],
```

Comment:
Do we want to keep the default options?

Reply:
The `options` prop is not used here, so I can probably just remove it.